### PR TITLE
get_gen() now returns a list

### PR DESCRIPTION
**Pull request by KevCaz**
Now `get_gen()` always returns a list of 2 elements: `body` and `response` that includes all the entries returned by the API (no matter what the actual number of pages is). Output formats are handled separately and currently: 
- we use objects of class `sf` for entries of `network` and `interaction`;
- objects of class `data.frame` for entries of the other endpoints.

### CODE REVIEW

File: R/search_taxa.R [hunk 1]
```diff
@@ -25,63 +25,77 @@
...
+  if (verbose)
+    message(sprintf("Found %s taxa involved in %s network(s)", nrow(taxa), length(network_ids)))
 
-    class(taxa) <- append(class(taxa), "mgSearchTaxa")
-    taxa
+  class(taxa) <- append(class(taxa), "mgSearchTaxa")
+  taxa
 
 }
+
```
Comment (SteveViss) [inline R/search_taxa.R:79]: This comment is not needed anymore.

File: R/zzz.R [hunk 2]
```diff
@@ -143,107 +207,37 @@ get_gen <- function(endpoint, query = NULL, limit = 100, flatten = TRUE,
 #' @details
 #' See endpoints available with `endpoints()`
 #' @keywords internal
-
-get_singletons <- function(endpoint = NULL, ids = NULL, output = "data.frame",
-flatten = TRUE, verbose = FALSE,...) {
+# get_singletons(endpoint = endpoints()$network, ids = c(1101, 1102), flatten = FALSE)
```
Comment (SteveViss) [inline R/zzz.R]: Not needed.

**Review by SteveViss — COMMENTED** [review]
Awsome job !

**Comment by KevCaz** [comment]
Je vais enlever flatten et les trucs inutiles.

**Review by SteveViss — APPROVED** [review]

**Pull request merged**